src/util/util_string.cpp

```diff
@@ -15,16 +19,26 @@ namespace dxvk::str {
     ::WideCharToMultiByte(CP_UTF8, 0, ws, -1,
       &result.at(0), len, nullptr, nullptr);
     return result;
+#else
+    std::wstring_convert<std::codecvt_utf8<wchar_t>> converter;
```

Comment:
This is deprecated in C++17.

Reply:
I hate the C++ standards committee. *sigh*

Reply:
Wow they really deprecated this and said "no replacement" huh.

Absolutely incredible work.